treeherder/log_parser/artifactbuilders.pyx

```diff
@@ -98,3 +108,64 @@ class BuildbotPerformanceDataArtifactBuilder(ArtifactBuilderBase):
             TalosParser()
         ]
         self.name = "talos_data"
+
+class MozlogArtifactBuilder(ArtifactBuilderBase):
+    """Extracts a summary artifact from a Mozlog log"""
+    def __init__(self, url=None):
+        """Construct artifact builder for the log viewer"""
+        super(MozlogArtifactBuilder, self).__init__(url)
+        self.name = "json_log_summary"
+        self.url = url
+
+    class SummaryHandler(object):
+        def __init__(self):
+            self.serial = 0
+            self.lines = []
+
+        def is_fault(self, message):
+            try:
+                retval = any(["level" in message and
```

Comment:
Perhaps s/retval/result/ or even just returning on the any() without assigning first?
(To me, retval implies the exit code from launching a process)